pkgs/top-level/aliases.nix

```diff
@@ -1225,11 +1225,13 @@ mapAliases ({
   ;
 
   # LLVM packages for (integration) testing that should not be used inside Nixpkgs:
-  llvmPackages_git = recurseIntoAttrs (callPackage ../development/compilers/llvm/git {
+  llvmPackages_git = recurseIntoAttrs (callPackage ../development/compilers/llvm/git ({
     inherit (stdenvAdapters) overrideCC;
     buildLlvmTools = buildPackages.llvmPackages_git.tools;
     targetLlvmLibraries = targetPackages.llvmPackages_git.libraries;
-  });
+  } // lib.optionalAttrs stdenv.hostPlatform.isDarwin {
+    inherit (llvmPackages_11) stdenv;
+  }));
```

Comment:
This change isn't present in the PR for staging. It is necessary here because the Darwin stdenv in staging-21.11 uses LLVM 7, which refuses to compile libcxx from LLVM 14.